2021-11-22 22:12:37,792 INFO    MainThread:24918 [wandb_setup.py:_flush():69] setting env: {}
2021-11-22 22:12:37,792 INFO    MainThread:24918 [wandb_setup.py:_flush():69] setting login settings: {}
2021-11-22 22:12:37,792 INFO    MainThread:24918 [wandb_init.py:_log_setup():338] Logging user logs to /home/workspace/FedML/fedml_experiments/standalone/fedavg/wandb/run-20211122_221237-37xpbixg/logs/debug.log
2021-11-22 22:12:37,792 INFO    MainThread:24918 [wandb_init.py:_log_setup():339] Logging internal logs to /home/workspace/FedML/fedml_experiments/standalone/fedavg/wandb/run-20211122_221237-37xpbixg/logs/debug-internal.log
2021-11-22 22:12:37,792 INFO    MainThread:24918 [wandb_init.py:init():371] calling init triggers
2021-11-22 22:12:37,793 INFO    MainThread:24918 [wandb_init.py:init():378] wandb.init called with sweep_config: {}
config: {'model': 'resnet18_gn', 'dataset': 'fed_cifar100', 'data_dir': './../../../data/fed_cifar100/datasets', 'partition_method': 'hetero', 'partition_alpha': 0.5, 'batch_size': 10, 'client_optimizer': 'sgd', 'lr': 0.1, 'wd': 0.001, 'epochs': 1, 'client_num_in_total': 10, 'client_num_per_round': 10, 'comm_round': 4000, 'frequency_of_the_test': 5, 'gpu': 0, 'ci': 0}
2021-11-22 22:12:37,793 INFO    MainThread:24918 [wandb_init.py:init():420] starting backend
2021-11-22 22:12:37,793 INFO    MainThread:24918 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-22 22:12:37,806 INFO    MainThread:24918 [backend.py:ensure_launched():135] starting backend process...
2021-11-22 22:12:37,811 INFO    MainThread:24918 [backend.py:ensure_launched():140] started backend process with pid: 24976
2021-11-22 22:12:37,812 INFO    MainThread:24918 [wandb_init.py:init():425] backend started and connected
2021-11-22 22:12:37,815 INFO    MainThread:24918 [wandb_init.py:init():473] updated telemetry
2021-11-22 22:12:37,816 INFO    MainThread:24918 [wandb_init.py:init():492] communicating current version
2021-11-22 22:12:38,537 INFO    MainThread:24918 [wandb_init.py:init():497] got version response upgrade_message: "wandb version 0.12.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-11-22 22:12:38,537 INFO    MainThread:24918 [wandb_init.py:init():505] communicating run to backend with 30 second timeout
2021-11-22 22:12:38,787 WARNING MainThread:24918 [wandb_init.py:init():782] interrupted
Traceback (most recent call last):
  File "/home/cps01/anaconda3/envs/FEDML/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 762, in init
    run = wi.init()
  File "/home/cps01/anaconda3/envs/FEDML/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 506, in init
    ret = backend.interface.communicate_run(run, timeout=30)
  File "/home/cps01/anaconda3/envs/FEDML/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 666, in communicate_run
    return self._communicate_run(run, timeout=timeout)
  File "/home/cps01/anaconda3/envs/FEDML/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 654, in _communicate_run
    resp = self._communicate(req, timeout=timeout)
  File "/home/cps01/anaconda3/envs/FEDML/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 544, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/cps01/anaconda3/envs/FEDML/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 82, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/cps01/anaconda3/envs/FEDML/lib/python3.6/threading.py", line 551, in wait
    signaled = self._cond.wait(timeout)
  File "/home/cps01/anaconda3/envs/FEDML/lib/python3.6/threading.py", line 299, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
